@navikt/aksel-icons/package.json

```diff
@@ -55,7 +59,8 @@
     "parse-svg": "node -e 'require(\"./config/parse-svg.js\").main()'",
     "copy": "copyfiles util/* src",
     "update-metadata": "node config/metadata.js",
-    "build": "yarn copy && yarn create-icons && concurrently \"tsc\" \"tsc -p tsconfig.esm.json\" && node config/cleanTypes.js && yarn update-metadata && yarn parse-svg",
+    "write-packagejson": "echo '{\"type\": \"module\"}' > dist/react/esm/package.json",
```

Comment:
Trengs for at filer under `/esm` skal tolkes som ESM-filer. Alternativet ville vært at alle filer i `/esm` får `.mjs`-extension